fmriprep/workflows/epi.py

```diff
@@ -728,6 +738,11 @@ def init_func_reports_wf(reportlets_dir, freesurfer, name='func_reports_wf'):
         DerivativesDataSink(base_directory=reportlets_dir,
                             suffix='tcompcor'),
         name='ds_tcompcor_report', run_without_submitting=True)
+    if use_aroma:
+        ds_ica_aroma_report = pe.Node(
+            DerivativesDataSink(base_directory=reportlets_dir,
+                                suffix='ica_aroma'),
+            name='ds_ica_aroma_report', run_without_submitting=True)
```

Comment:
You can create the node unconditionally. The cost of creating the node and then not connecting it is miniscule, and avoiding unnecessary branching makes reasoning easier.